docs/usage/key-concepts/presets.md

```diff
@@ -32,6 +46,12 @@ Renovate comes with a lot of built-in presets that you can use.
 Browse [Renovate's default presets](https://docs.renovatebot.com/presets-default/) to find any that are useful to you.
 Once you find a preset you like, put it in an `extends` array in your config file.
 
+### Contributing a new built-in preset
```

Comment:
I just added a small section in this PR to explain that users can contribute their configuration.

The section I'm linking to says:

> Have you configured a rule that you think others might benefit from? Please consider contributing it to the [Renovate repository](https://github.com/renovatebot/renovate/tree/main/lib/config/presets/internal) so that it gains higher visibility and saves others from reinventing the same thing.

Both sections do not actually explain _how_ to get the user's config into a Renovate preset. Do we want to list steps like:

1. Criteria for presets that we'll generally accept so user's don't waste time on things that will be rejected
1. Find relevant part of the preset (usually `group` or `monorepo`)
1. Explain when to choose `group` and when to pick `monorepo`
1. Open PR and get reviewed

Reply:
I'd direct them to create a discussion to propose